src/api/json/catalog.json

```diff
@@ -302,7 +302,10 @@
       "name": "sarif.json",
       "description": "Static Analysis Results Interchange Format (SARIF)",
       "fileMatch": [ "*.sarif,*.sarif.json" ],
-      "url": "http://json.schemastore.org/sarif"
+      "url": "http://json.schemastore.org/sarif",
+      "versions": {
+        "1.0.0": "http://json.schemastore.org/sarif-1.0.0"
```

Comment:
Perfect; that's what Michael and I thought but we wanted to be sure. In short:

1. A client that does a GET request on http://json.schemastore.org/sarif-1.0.0 will get the file src/schemas/json/sarif-1.0.0.json, simply because that latter file exists -- regardless of anything that might or might not be in the catalog.

2. A client that knows how to make an API request for "SARIF v1.0.0" would consult your catalog, find the entry `1.0.0` under `versions`, and retrieve the same file.